data/kahet/kahet missions.txt

```diff
@@ -721,3 +721,1159 @@ event "pacified ka'sei"
+			choice
+				`	"It is. Let's go."`
+			`	You help Dusk transport the chosen instruments to your ship; even if, on their own, they only occupy a small portion of the <ship>'s cargo bay, you also make sure to leave enough space for the facilities you're going to take from <stopovers>. As you're walking back toward the ship's controls, Dusk reminds you to bring as many ramscoops with you as possible - you're going to need them in order to complete the mission plan while cloaked.`
+				accept
+	on stopover
+		dialog
+			`Dusk's coordinates indicate the border of a large canyon near the equator of <planet>. Unlike most cracks on the surface, this one shows signs of geological activity: you only narrowly avoid a stream ejecting from a geyser in the canyon as you pass overhead. As you land, a row of Remnant structures are clearly visible running parallel to the cliff.`
+			`	The local crew assist in loading the modular laboratory split across three pallets; with them arrives a separate package, containing a surveillance satellite to be deployed near Patir to scan the Ka'sei from orbit.`
+	on fail
+		dialog
+			`The Ka'sei saw your ship and started following you, making the exploration impossible. You failed the mission.`
+	on enter Patir
+		dialog
+			`The probe slowly moves away from your ship, opening its external panels and the instrumentation; as you watch it turn into a dim point of light, the Remnant team signals you it's functioning correctly, and you prepare to approach the asteroid.`
+
+	npc save
+		government "Test Dummy"
+		personality unconstrained heroic nemesis
+		system "Patir"
+		fleet
+			variant
+				"Cloak Check" 10
+
+mission "Ka'het: Patir Mystery 9"
+	landing
+	name "Explore the neighborhood"
+	description "Visit every system around Patir you can reach, then return to Dusk in <system>."
+	source "Builder Settlement"
+	waypoint "Athiri"
+	waypoint "Chanai"
+	waypoint "Ghila"
+	waypoint "Lathia"
+	waypoint "Maithi"
+	waypoint "Mitera"
+	waypoint "Sarifa"
+	waypoint "Thepa"
+	to offer
+		has "Ka'het: Patir Mystery 8A: done"
+	on offer
+		event "Patir moves"
+		conversation
+			`Approaching the settlement from northeast, you park the <ship> once again next to airlock array; it's almost starting to become a familiar sight by now. With the cargo ramp pointed toward the entrance and the post-flight checklist completed, the deployment of the lab can finally commence; the Remnant team disembarks quickly and starts working on the first pallet, while you remain in the cockpit keeping an eye on the Ka'sei activity in the area. The researchers work quickly and nearly without issues in the light gravity of the asteroid, besides the occasional lost screw, but the construction job still takes nearly the entire day.`
```

Comment:
The ship and lab are meant to be uncloaked on the terrain, as it would needlessly complicate the operations. After all, walking out of the ship would make them detectable anyways, and the cloak itself is only meant to trick the ka'sei into thinking nobody new is visiting the lab.

Added the suggestion separately btw, to avoid spamming the PR with commits